### DISCUSSION TITLE
2024/10/01/merge-lora-to-stable-diffusion-model/

### DISCUSSION THREAD

**Discussion by giscus** [licyk/licyk.github.io#22] [Comment]
# 2024/10/01/merge-lora-to-stable-diffusion-model/

一个简单的模型融合方法。

https://licyk.github.io/2024/10/01/merge-lora-to-stable-diffusion-model/

**Comment by handerson23333** [comment]
很有用

**Comment by AaaNuyoah** [comment]
博主你好，我将lora模型和sd1.5模型进行合并，能够输出模型，但是打印出来的信息有Unet2ConditionModel :64,8,768,Flase,Flase，这说明Unet2ConditionModel 部分合并有问题是吗？
